src/error.rs

```diff
@@ -1,9 +1,39 @@
-use std::io::{Error, ErrorKind};
-
-/// returns an error if a command is not provided
-pub fn error() -> Result<(), std::io::Error> {
-    Err(Error::new(
-        ErrorKind::InvalidInput,
-        "invalid input: please provide a command",
-    ))
+use std::fmt;
+use std::io;
+
+extern crate csv;
+
```

Comment:
You could add:

```
/// A type alias for `Result<T, Error>`.
pub type Result<T> = result::Result<T, Error>;
```

Which would allow you to change:
```
pub fn register(filename: &String, option: &String) -> Result<(), Error>
```
to
```
pub fn register(filename: &String, option: &String) -> Result 
```

Reply:
This a nice shortcut - thanks!